pkg/api/authz.go

```diff
@@ -286,17 +284,41 @@ func AuthzHandler(ctlr *Controller) mux.MiddlewareFunc {
 
 			ctx := acCtrlr.getContext(acCtx, request)
 
-			/* Notes:
-			 	- since we only do READ actions in extensions, we can bypass authz for them
-				only need to know the username, whether the user is an admin, or what repos he can read.
-				let each extension to apply	authorization on them using localCtx.AccessControlContext{}
-			*/
-			if isExtensionURI(request.RequestURI) {
-				next.ServeHTTP(response, request.WithContext(ctx)) //nolint:contextcheck
+			next.ServeHTTP(response, request.WithContext(ctx)) //nolint:contextcheck
```

Comment:
why the `nolint`

Reply:
this was already existing before on the main branch because the linter complains of non-inherited context. It is actually inherited, but not through the methods it expects - `context.WithXXX` or `r.Context`